ngraph/core/reference/include/ngraph/runtime/reference/region_yolo.hpp

```diff
@@ -0,0 +1,164 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include <cmath>
+
+#include "ngraph/shape.hpp"
+
+namespace ngraph
+{
+    namespace runtime
+    {
+        namespace reference
+        {
+            static inline int entry_index(int width,
+                                          int height,
+                                          int coords,
+                                          int classes,
+                                          int outputs,
+                                          int batch,
+                                          int location,
+                                          int entry)
+            {
+                int n = location / (width * height);
+                int loc = location % (width * height);
+                return batch * outputs + n * width * height * (coords + classes + 1) +
+                       entry * width * height + loc;
+            }
+
+            static inline float logistic_activate(float x) { return 1.f / (1.f + std::exp(-x)); }
```

Comment:
I think this should be called a sigmoid rather than logistic_activate.

Reply:
That's right. I changed it to sigmoid.